_i18n/en/documentation/podcasters-hosters/funding.md

```diff
@@ -0,0 +1,7 @@
+To ensure the sustainability of an open podcasting ecosystem, podcasters need funding beyond advertising and buy-outs by platforms. AntennaPod supports different ways through which podcasters can earn an income:
+
+* We support the [funding](https://podcasting2.org/docs/podcast-namespace/tags/funding) tag, allowing podcasters to add links to their payment pages in their RSS feeds: see [how to use this tag](https://podcasting2.org/docs/guides/how-to-use-the-funding-tag). This is the standard way for podcasters to get financial support from their audience.
+
+* If for any reason you don't use the funding tag yet, you can still include information about how your audience can support you (e.g. via a premium tier) in your podcast or episode descriptions. It will still be visible to AntennaPod users, even if less prominently.
```

Comment:
I think "even though" was correct here, it's there to show contrast (less prominent than the tag)